VERSION

```diff
@@ -1 +1 @@
-2.5.1.0
```

Comment:
Please revert this change to the `VERSION` file.. this is only updated by maintainers just before a release.

Reply:
I need you to call me asap this is Blackstone CYM my number is 2798677377 I got a M dollar prop. For you

Reply:
Tried to call whats the M dollar prop

Reply:
5 mill the money I have a proposition for that much in dollars